membership-attribute-service/app/services/AttributesMaker.scala

```diff
@@ -55,7 +56,9 @@ class AttributesMaker {
     }
 
     maybeAlertAvailable.getOrElse(Future.successful(None)) map { maybeAlert =>
-      maybeAlert map { alert => SafeLogger.info(s"User $identityId has an alert available: $alert") }
+      def customFields(identityId: String, alertAvailableFor: String) = List(LogFieldString("identity_id", identityId), LogFieldString("alert_available_for", alertAvailableFor))
```

Comment:
we already use camelcase elsewhere for identity id:
https://membership-logs.gutools.co.uk/app/kibana#/discover?_g=()&_a=(columns:!(identityId),index:AWAsTQ6MXSopLsUBjTYb,interval:auto,query:(match_all:()),sort:!('@timestamp',desc))

Reply:
Ooops, thanks for catching that.